modules/ocf_mirrors/templates/systemd/sync.service.erb

```diff
@@ -8,3 +8,4 @@ Type=simple
 Environment="<%= env %>=<%= val %>"
 <% end -%>
 ExecStart=<%= @exec_start %>
+TimeoutStartSec=86400 # one day should be good enough for anyone
```

Comment:
My understanding is that comments need to be on their own lines to be syntactically correct. Just put it above.

Reply:
nice. you are correct `system.syntax(7)`.